### DISCUSSION TITLE
posts/2017-05-14-why-georgia-tech-omscs/

### DISCUSSION THREAD

**Discussion by giscus** [zmsp/zmsp.github.io#3] [General]
# posts/2017-05-14-why-georgia-tech-omscs/

Why OMSCS (Online Master of Science in Computer Science) at Georgia Tech?

https://www.zobairshahadat.com/posts/2017-05-14-why-georgia-tech-omscs/

**Comment by preciousjerry** [comment]
I am impressed by the construct of your article. I've been looking for a good article about <a href="https://www.makeoverarena.com/zenni-affordable-glasses-sunglasses-for-everyone/">affordable glasses</a> when I stumbled on your website and it really brought my search to an end.

You should review;

https://www.makeoverarena.com/mp3juice-download-free-mp3-con-mp3juice-download-free-mp3-converter/
